plugins/inputs/kube_inventory/kube_inventory.go

```diff
@@ -140,6 +145,57 @@ func (ki *KubernetesInventory) convertQuantity(s string, m float64) int64 {
 	}
 	return int64(f * m)
 }
+func (ki *KubernetesInventory) LoadJSON(url string, v interface{}) error {
```

Comment:
Please do not export this function!
```suggestion
func (ki *KubernetesInventory) loadJSON(url string, v interface{}) error {
```